_gsdocs-orgs/2019/ral.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "STFC Rutherford Appleton Laboratory"
+author: "Benedikt Hegner"
+layout: default
+organization: RAL
+logo: logo_RAL.jpg
+description: |
+  STFC Rutherford Appleton Laboratory in Oxfordshire, United Kingdom is home to national facilities including the Diamond synchrotron and ISIS neutron spallation source. The Particle Physics Department contributes to experiments including ATLAS, CMS and T2K.
+---
+
+{% include gsoc_proposal.ext %}
```

Comment:
There is a new extension for Season of Docs proposals, please add the following change otherwise the `List of proposals` link will point to the GSoC projects:

```suggestion
{% include gsdocs_proposal.ext %}
```